src/Psalm/Internal/Analyzer/Statements/Expression/BitwiseNotAnalyzer.php

```diff
@@ -59,6 +60,7 @@ public static function analyze(
                         new TLiteralInt(~$type_part->value) :
                         new TInt;
 
+                    // todo:immutable this makes no sense?
```

Comment:
is this a part of what you intend to fix in a next PR?

Reply:
Actually now that I look at it, that code makes perfect sense, removed the comment :)